docs/src/modules/components/PropertiesTable.js

```diff
@@ -88,7 +88,7 @@ export default function PropertiesTable(props) {
                 .filter((key) => propData.additionalInfo?.[key])
                 .map((key) => (
                   <p
-                    className="prop-list-additional-info"
+                    className="prop-list-description"
```

Comment:
| Before | After |
| ------ | ----- |
| <img width="725" src="https://github.com/mui/material-ui/assets/13808724/10b3fee5-b769-446d-996f-da1ab116d7f1"> | <img width="692" src="https://github.com/mui/material-ui/assets/13808724/bd03dc5f-30fc-4bc4-a2ac-7aa3c3bb6d60"> |

Reply:
Would it be possible to add this content in an `info` callout?
Additionally, it'd be great to render content wrapped in backticks as a `code` block!

Reply:
Nit-picky: how about putting "_the_" in the link? Like:
> See [the `sx` page](https://mui.com/system/getting-started/the-sx-prop/) for more details.

Reply:
Not sure about the info callout. It's the kind of text you read once, then you can ignore it. 

It's for new users encountering this kind of props for the first time, they got the link to the docs just next to the description they did not understand.

Reply:
Yes, those wordings did not get attention for a long time. They could get some reviews from @samuelsycamore all together at once

[Here](https://github.com/mui/material-ui/blob/master/docs/src/modules/components/PropertiesTable.js/#L8-L19) you can find all the API descriptions added automatically